src/client/components/ReferenceAutocomplete/i18n/de.js

```diff
@@ -1,9 +1,11 @@
 let ReferenceAutocomplete = {
   clearValueText: "Löschen",
   clearAllText: "Alles löschen",
-  noResultsText: "Kein Suchergebnis. Bitte ändern Sie die Suchkriterien und versuchen Sie es erneut.",
+  noResultsText: "No matches found",
   placeholder: "Werte hinzufügen",
-  loadingPlaceholder: "Bitte warten..."
+  loadingPlaceholder: "Searching...",
+  loadingError: "Loading failed",
+  notEnoughCharacters: "Please enter {minAutocompleteChars} or more characters"
```

Comment:
Please, check it to German `Bitte geben Sie {minAutocompleteChars} oder mehr Zeichen ein`